File: backend/src/main/java/com/gymapp/modules/training/dto/WorkoutLogResponse.java
package com.gymapp.modules.training.dto;

import lombok.Builder;
import lombok.Data;
import lombok.Getter;
import lombok.Setter;
import java.time.LocalDate;
import java.time.OffsetDateTime;
import java.util.UUID;

@Data
@Getter
@Setter
@Builder
public class WorkoutLogResponse {
    private UUID id;
    private UUID planId;
    private String planName;
    private LocalDate logDate;
    private Integer durationMinutes;
    private String notes;
    private boolean completed;
    private OffsetDateTime createdAt;
}

File: android/app/src/main/java/com/gymapp/android/data/remote/dto/WorkoutLogResponse.kt
package com.gymapp.android.data.remote.dto

data class WorkoutLogResponse(
    val id: String,
    val planId: String?,
    val planName: String?,
    val logDate: String,
    val durationMinutes: Int?,
    val notes: String?,
    val completed: Boolean,
    val createdAt: String?
)